metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -190,6 +190,14 @@ object ServerCommands {
     "[string], where the string is a stacktrace."
   )
 
+  val Decompile = new Command(
+    "decompile",
+    "Show TASTy",
+    """|Show tasty file and show its content.
```

Comment:
```suggestion
    """|If the file is a Scala 3 source  this command will try to find the relevant tasty file for it, read it and display in a human readable form. If the argument already points to a Scala file, it will be read directly.
```